plugin/storage/integration/fixtures/queries_es.json

```diff
@@ -0,0 +1,34 @@
+[
+    {
+        "Caption": "Tag excaped operator + Operation name + max Duration",
+        "Query": {
+        "ServiceName": "query23-service",
+        "OperationName": "query23-operation",
+        "Tags": {
+            "sameplacetag1":"random\\*"
```

Comment:
Can we use `same\\*`. 

The other fixtures contain `sameplacetag1: sameplacevalue` so we should see it does not bring other results.